src/basis/basis.js

```diff
@@ -440,6 +440,42 @@ export class Basis {
         return math.precise(latticeSize, 4);
     }
 
+    /**
+     * @summary function returns an array of overlapping atoms within specified tolerance.
+     * @returns {{element: String, id: Number}[]}
+     */
+    getOverlappingAtoms() {
```

Comment:
We should make this configurable and turn off when the number of atoms exceeds a certain value - i.e. 1000